src/components/Executions/ExecutionDetails/ExecutionMetadata.tsx

```diff
@@ -23,7 +23,7 @@ const useStyles = makeStyles((theme: Theme) => {
       position: 'relative',
     },
     detailsContainer: {
-      alignItems: 'center',
+      alignItems: 'start',
```

Comment:
nit: Maybe it's only me, but I feel like icon should go 1-2 px lower, to align better with text (as it mainly uses lower case naming), plus it looks like in cases when cache icon present right margin after the icon should become smaller for 2px too.
<img width="439" alt="Screen Shot 2022-04-18 at 12 32 29 PM" src="https://user-images.githubusercontent.com/55718143/163865773-3303464b-c4ef-453b-b5a4-2bd53e9cfbf2.png">

Reply:
added `1px` margin top in `customNodeComponents.tsx`, where the icon styles are defined